topics/ui/controls/link.md

```diff
@@ -10,127 +10,266 @@
+<table style="none" border="false">
+  <tr>
+    <td>
+      <format color="369650" style="bold">Correct</format>
+      <img src="link_how_to_2_correct.png" width="378" alt="A vertical list of three options: 'Non-Project Files', 'Problems', and 'Production', followed by a 'Manage scopes…' link below. The link has an ellipsis in the end."/>
+    </td>
+    <td>
+      <format color="E55765" style="bold">Incorrect</format>
+      <img src="link_how_to_2_incorrect.png" width="378" alt="Vertical list of three options: 'Non-Project Files', 'Problems', and 'Production', followed by a 'Manage scopes' link below. The link doesn't have an ellipsis in the end."/>
+    </td>
+  </tr>
+</table>
+
+#### Wording
+
+Do not use words like <code>navigate</code> or <code>click here</code>. A link already implies navigation or clicking.
+
+<table style="none" border="false">
+  <tr>
+    <td>
+      <format color="369650" style="bold">Correct</format>
+      <img src="link_how_to_3_correct.png" width="378" alt="Link 'Proxy settings…'."/>
+    </td>
+    <td>
+      <format color="E55765" style="bold">Incorrect</format>
+      <img src="link_how_to_3_incorrect.png" width="378" alt="Link 'Navigate to general proxy settings'."/>
+    </td>
+  </tr>
+</table>
+
+<table style="none" border="false">
+  <tr>
+    <td>
+      <format color="369650" style="bold">Correct</format>
+      <img src="link_how_to_4_correct.png" width="378" alt="Link 'Download drivers…'."/>
+    </td>
+    <td>
+      <format color="E55765" style="bold">Incorrect</format>
+      <img src="link_how_to_4_incorrect.png" width="378" alt="Text reading 'Click here to download drivers' where 'here' is a link."/>
+    </td>
+  </tr>
+</table>
+
+#### Make it informative
+
+When possible, replace <code>Learn more</code> and other generic phrases with more informative ones.
+
+<table style="none" border="false">
+  <tr>
+    <td>
+      <format color="369650" style="bold">Correct</format>
+      <img src="link_how_to_5_correct.png" width="378" alt="Link 'How to use Closure Linter'."/>
+    </td>
+    <td>
+      <format color="E55765" style="bold">Incorrect</format>
+      <img src="link_how_to_5_incorrect.png" width="378" alt="Link 'Learn more'."/>
+    </td>
+  </tr>
+</table>
 
-![](04_external_link.png){width=248}
-<tabs group="languages">
-<tab title="Kotlin" group-key="kotlin">
+### Link as a part of text
 
-```kotlin
-val externalLink = ActionLink("External link") {
-  BrowserUtil.browse("https://www.jetbrains.com")
-}.apply {
-  setExternalLinkIcon()
-}
-```
+A link can be a part of a checkbox or radio button label or of any stand-alone text, like the text in an [empty state](empty_state.md) or in [context help](context_help.md):
 
-</tab>
-<tab title="Java" group-key="java">
+![Checkbox 'Use color scheme instead of the deafult' where 'deafult' is a link.](link_how_to_6.png){width=706}
```

Comment:
The checkbox text is somewhat unclear here, I'd suggest using the one from Settings | Editor | Color Scheme | Console Font: "Use console font instead of the default font" (let's add "font" in the end to make it more clear in this example)